client/src/pages/EventPage.js

```diff
@@ -14,11 +15,21 @@ const Container = styled.div`
 `;
 
 function EventPage() {
+  const history = useHistory();
+  const [query, setQuery] = useState("");
   const { data: events } = useAsync(getEvents);
-  console.log(events);
+  function handleSubmit(event) {
+    event.preventDefault();
+    history.push(`/search?q=${query}`);
+  }
+
   return (
     <Container>
-      <Searchbar></Searchbar>
+      <Searchbar
+        value={query}
+        onChange={(event) => setQuery(event.target.value)}
+        onSubmit={handleSubmit}
```

Comment:
You could think of setting the state and defining the function in your component itself for a better re-use purpose